src/core/services/theming/theming.js

```diff
@@ -689,6 +758,40 @@ function generateAllThemes($injector, $mdTheming) {
     delete palette.contrastStrongLightColors;
     delete palette.contrastDarkColors;
 
+    function getContrastType(hueName) {
+      if (defaultContrast === 'light') {
+        if (darkColors.indexOf(hueName) > -1) return 'dark';
```

Comment:
That's funny because I normally use explicit braces and used inline because I saw it being using elsewhere in our code.